### ISSUE TITLE
Missing CRC checks

### ISSUE THREAD

**Issue by jake-g00dwin** [Personal-Data-Acquisition/sensor_lib_aht20#10]
Currently driver is missing the CRC for error checking of the data bytes received over I2C.

- [x] Add tests for CRC
- [x] Create module for CRC & sensor data
- [x] Write code to pass tests.

**Comment by jake-g00dwin** [comment]
So far so good, I've built out the tests as stubs and am currently working on building a detailed test for CRC8 calculations. Once this is done the module can be merged into the main repo.

**Comment by jake-g00dwin** [comment]
The CRC-8-MAXIM is now implemented and tested.
